doc/decisions/1_in_discussion/copy_on_write.md

```diff
@@ -12,9 +12,6 @@ There are also aspirations to create a new, simple [internal cache](../3_decided
 
 ## Constraints
 
-1. Elektra's public API should require no changes
```

Comment:
I think we should add an constraint that the life-time of KeySets also must be as expected. (And not be related to kdbClose).

Reply:
Again, what is "expected"?

I'd say: "The liftime of a `Key` and a `KeySet` must be unaffected by COW".